pkgs/applications/networking/instant-messengers/ferdium/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, mkFranzDerivation, fetchurl, xorg, xdg-utils, buildEnv, writeShellScriptBin }:
+
+let
+  mkFranzDerivation' = mkFranzDerivation.override {
+    xdg-utils = buildEnv {
+      name = "xdg-utils-for-ferdi";
+      paths = [
+        xdg-utils
+        (lib.hiPrio (writeShellScriptBin "xdg-open" ''
+          unset GDK_BACKEND
```

Comment:
Why is this necessary? Looks to me that instead of unsetting here, you should instead wrapper the `ferdium` derivation instead.

Reply:
This is another thing I just copied over from the ferdi package, but can't really see why it is there, though I assume the ferdi packagers did this for a reason. When I remove the whole `mkFranzDerivation'` and replace it with the normal `mkFranzDerivation` the package still builds fine. Should I remove it then?

Reply:
If this is also done in the Franz derivation you can probably just add this directly to the `mkFranzDerivation` function instead of duplicating the code here.

Reply:
It doesn't seem to be in `mkFranzDerivation`. The binary seems to work fine with `GDK_BACKEND=wayland` and `GDK_BACKEND=x11` (which is what this variable should be) and runs fine in wayland. I'll just remove the whole `mkFranzDerivation'` and move to `mkFranzDerivation` it and if someone has any problem with it they can open an issue.